sensor/common/virtualmachine/component.go

```diff
@@ -0,0 +1,27 @@
+package virtualmachine
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/sensor/common"
+)
+
+// Component provides functionality to send virtual machines to Central.
+type Component interface {
```

Comment:
[nit] Maybe VMHandler or just handler? that's how we call these type of sensor components in other parts.

Reply:
done in c62333e7060871c10987dded6bcb7ff78cd343a8